go/proto/ctrl/v1/service.proto

```
@@ -0,0 +1,17 @@
+syntax = "proto3";
+
+package ctrl.v1;
```

Buf (1.54.0)

Files with package "ctrl.v1" must be within a directory "ctrl/v1" relative to root but were in directory "go/proto/ctrl/v1".

(PACKAGE_DIRECTORY_MATCH)